pkg/mapper/testworkflows/kube_openapi.go

```diff
@@ -522,10 +522,58 @@ func MapPodResourceClaimKubeToAPI(v corev1.PodResourceClaim) testkube.PodResourc
 
 func MapPodSecurityContextKubeToAPI(v corev1.PodSecurityContext) testkube.PodSecurityContext {
 	return testkube.PodSecurityContext{
-		RunAsUser:    MapInt64ToBoxedInteger(v.RunAsUser),
-		RunAsGroup:   MapInt64ToBoxedInteger(v.RunAsGroup),
-		RunAsNonRoot: MapBoolToBoxedBoolean(v.RunAsNonRoot),
-		FsGroup:      MapInt64ToBoxedInteger(v.FSGroup),
+		SeLinuxOptions:           common.MapPtr(v.SELinuxOptions, MapSELinuxOptionsKubeToAPI),
```

Comment:
`openapi_kube` definition (reverse) is missing too 👍

Reply:
added